infra/prod/italynorth/locals.tf

```diff
@@ -0,0 +1,19 @@
+locals {
+  prefix               = "io"
+  env_short            = "p"
+  location_short       = "itn"
+  location             = "italynorth"
+  project              = "${local.prefix}-${local.env_short}-${local.location_short}"
+  application_basename = "services"
+  srch_snet_cidrs      = ["10.20.135.0/26"]
```

Comment:
how did you get this value?

Reply:
With the aim of introducing a CIDR allocation strategy in the near future, we've opted for a similar IP class to the one employed by the project in the west Europe region (10.0.135.*/26).

For the other values, instead, I took inspiration from the work done by @Krusty93 when implementing the [IaC for io-infra Italy North region](https://github.com/pagopa/io-infra/tree/main/src/core/prod/italynorth).